source/reference/metrics.txt

```diff
@@ -0,0 +1,269 @@
+.. _app-services-metrics:
+
+==============================
+App Services Metrics Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+App Services provides a variety of metrics related to App Services and Device
+Sync. You can access your metrics data with the Admin API's
+:admin-api-endpoint:`getMetrics <tag/metrics/operation/adminGetMetrics>` endpoint.
+
+The exact metrics returned may be subject to change.
+
+.. Source derived from https://docs.google.com/spreadsheets/d/1cjQopjPUsEKde6TlqO38B0Ko6n4vTwl4tgJ-LHSbf7Y/edit#gid=1354315462
+
+Units
+-----
+
+App Services expresses each metric in one of the following units:
+
+- BYTES_PER_SECOND
+- MILLISECONDS
+- SCALAR
+- SCALAR_PER_SECOND
+
+A **scalar** is a singular value representing a magnitude or count. Its exact
+interpretation depends on the metric.
+
+A **scalar per second** represents a count over time ("rate"). The value is
+effectively the total count in the requested time window divided by the number
+of seconds in the window.
+
+Some scalar per second values are expressed as a 95th percentile. This means
+that the reading for the given metric is below the given value 95% of the time
+and above it the other 5% of the time.
+
+Available Metrics
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 20 30
+
+  * - Metric Name
+    - Unit
+    - Description
+
+  * - ACTIVE_OPEN_SYNC_SESSIONS
+    - SCALAR
+    - The number of active, open sync sessions at the time of the reading.
```

Comment:
what do you mean by "time of reading"? is this not using the query params for `start` and `end`?

Reply:
> If the interval is like a day, is it the number of sessions open that day or is it a snapshot of the exact time of measurement? Not quite sure about this.

i think this is an important point to clarify.

> I think throwing in the word "reading" was a mistake. I can call it "measurement". Not sure if that alone would clarify anything but at least I wouldn't be using a new term. (FWIW, I chose the former because it sounded less stuffy but I changed my mind looking at it now.)

i'm partial to measurements..but maybe i'm just stuffy. defer to you on phrasing there. i think the above clarification point is what must be clarified for this PR.